media-proxy/src/api_server_tcp.cc

```diff
@@ -134,63 +136,9 @@ void* msg_loop(void* ptr)
                 if (it->id == session_id) {
                     /* return memif parameters. */
                     memif_conn_param param = { };
-                    if (it->type == TX) {
```

Comment:
beautiful :tada: I was devastated every time I looked at that spaghetti. good job